mmedit/models/restorers/tdan.py

```diff
@@ -0,0 +1,143 @@
+import numbers
+import os.path as osp
+
+import mmcv
+
+from mmedit.core import tensor2img
+from ..builder import build_loss
+from ..registry import MODELS
+from .basic_restorer import BasicRestorer
+
+
+@MODELS.register_module()
+class TDAN(BasicRestorer):
+    """TDAN model for video super-resolution.
+
+    Paper:
+        TDAN: Temporally-Deformable Alignment Network for Video Super-
+        Resolution, CVPR, 2020
+
+    Args:
+        generator (dict): Config for the generator structure.
+        pixel_loss (dict): Config for pixel-wise loss.
+        train_cfg (dict): Config for training. Default: None.
+        test_cfg (dict): Config for testing. Default: None.
+        pretrained (str): Path for pretrained model. Default: None.
+    """
+
+    def __init__(self,
+                 generator,
+                 pixel_loss,
+                 lr_pixel_loss,
```

Comment:
There is a mixed usage of lr vs lq. If they are same, then we can use one of them (lq)